common/lib/xmodule/xmodule/modulestore/mixed.py

```diff
@@ -172,8 +172,12 @@ def _get_modulestore_for_courseid(self, course_id=None):
             if mapping is not None:
                 return mapping
             else:
+                if isinstance(course_id, LibraryLocator):
```

Comment:
Now that a `course_id` can refer to `LibraryLocator` or `CourseLocator` since we're storing both courses and problem libraries in the modulestores as separate entities, it's unfortunate that we still use `course_id` to refer to both types of locators - it's confusing. Though - I can't think of a better term at the moment.